<ui version="4.0" >
 <class>cheatFinder</class>
 <widget class="QDialog" name="cheatFinder" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Cheat Finder</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="7" >
    <widget class="QTextEdit" name="textEdit" >
     <property name="html" >
      <string>Results</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QLineEdit" name="lineEdit" >
     <property name="text" >
      <string>Value</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="3" >
    <widget class="QPushButton" name="FindCheatValue" >
     <property name="text" >
      <string>Find Value</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QPushButton" name="FindCheatLess" >
     <property name="text" >
      <string>&lt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QPushButton" name="FindCheatLessEqual" >
     <property name="text" >
      <string>&lt;=</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QPushButton" name="FindCheatEqual" >
     <property name="text" >
      <string>==</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" >
    <widget class="QPushButton" name="FindCheatNotEqual" >
     <property name="text" >
      <string>!=</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4" >
    <widget class="QPushButton" name="FindCheatMoreEqual" >
     <property name="text" >
      <string>=></string>
     </property>
    </widget>
   </item>
   <item row="2" column="5" >
    <widget class="QPushButton" name="FindCheatMore" >
     <property name="text" >
      <string>></string>
     </property>
    </widget>
   </item>
   <item row="2" column="6" >
    <widget class="QPushButton" name="FindCheatRestart" >
     <property name="text" >
      <string>New Search</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
